api/pom.xml

```diff
@@ -296,6 +296,16 @@
             <artifactId>jackson-dataformat-xml</artifactId>
             <version>${jackson.version}</version>
         </dependency>
+        <dependency>
```

Comment:
We did a lot of exploring in this change. Do we actually need these new dependencies still? Let's verify.

Reply:
I dont think we need it anymore